zendesk/user.go

```diff
@@ -256,7 +259,35 @@ func (z *Client) GetManyUsers(ctx context.Context, opts *GetManyUsersOptions) ([
 	return data.Users, data.Page, nil
 }
 
-//TODO: GetUsersByGroupID, GetUsersByOrganizationID
+// GetUsersByGroupID get users by group ID
+// ref: https://developer.zendesk.com/api-reference/ticketing/users/users/#list-users
+func (z *Client) GetUsersByGroupID(ctx context.Context, groupID int64, opts *UserListOptions) ([]User, Page, error) {
```

Comment:
This is still using the old OBP UserListOptions and this function shouldn't be exist, it should be replaced by the newly generated GetGroupUsersIterator function.